kotlin/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM iron/java
+
+ENV VERSION 1.0.5
+
+ENV KOTLIN_HOME=/usr/share/kotlin
+ENV KOTLIN_COMPILER_URL https://github.com/JetBrains/kotlin/releases/download/v${VERSION}/kotlin-compiler-${VERSION}.zip
+
+RUN apk update
+RUN apk add bash
```

Comment:
Great. I didn't want to use 'apk upgrade'. It resulted in a JVM initialization error: 

> Invalid layout of java.lang.Thread at name

Something to watch out for.